### ISSUE TITLE
Improve search algorithm

### ISSUE THREAD

**Issue by nmorduch** [newamericafoundation/newamerica-cms#1426]
Split from #1414, especially https://github.com/newamericafoundation/newamerica-cms/issues/1414#issuecomment-546925768

- [x] **Disable `partial_match`**
  This is a feature of Wagtail that allows any word in the query to match on the prefix of any word in the title. It is on by default in Wagtail but only makes sense for autocomplete searches. We should switch it off for actual searches though.
 

- [x] **Take document age into account**
  If the site uses Elasticsearch, we can customise the query so that older results have their score lowered to give preference to newer results.

**Comment by AnnaGrieve** [comment]
Jacob to review

**Comment by AnnaGrieve** [comment]
Change boosting on title in ranking - see if this works

**Comment by nmorduch** [comment]
Opened a new PR. @kaedroho can you take a look at it and see what you think?

Can we define some page types as not showing up in search at all, and can we boost page types?
The search exposes some pages that shouldn't show up ever, e.g. https://www.newamerica.org/new-america-subscription-lists/education-policy-newsletters-events/

**Comment by nmorduch** [comment]
Removing the estimate since it's old and I'm not even sure how much more we want to change this.

**Comment by kaedroho** [comment]
> Can we define some page types as not showing up in search at all, and can we boost page types?

Yep, we can completely remove page types. Boosting is a bit trickier though since the title field is shared across all page types and there's not a way to boost individual documents. But we could probably do it as part of the query.

**Comment by nmorduch** [comment]
Going to wait on #1476
